bin/picca_export.py

```diff
@@ -240,6 +241,44 @@ def main(cmdargs):
         'R-parallel', 'R-transverse', 'Redshift', 'Correlation',
         'Covariance matrix', 'Distortion matrix', 'Number of pairs'
     ]
+
+    # Check if we need blinding and apply it
+    if 'BLIND' in data_name or blinding != 'none':
```

Comment:
Why not put `if blinding == "corr_yshift"`? Otherwise add a TODO saying to change this once we are ready to unblind

Reply:
Yeah this has to do with erring on the safe side (blind as much as possible). When unblinding we would would run picca_deltas without the blinding. So the change will be in picca_deltas not here, and this code doesn't get used at that point. But let's talk about this tomorrow.

Reply:
We could also add an 
```
# TODO: remove this when we are ready to unblind
else:
    raise ValueError("Expected blinding strategy corr_yshift")
```
but let's discuss it tomorrow morning

Reply:
I think it is ok as it is now, since it allows you to run picca_export.py on correlations function files that were computed using the minimal strategy, and it blinds them with the new strategy corr_yshift